scripts/download.py

```diff
@@ -26,15 +31,38 @@ def download_from_hub(repo_id: Optional[str] = None, access_token: Optional[str]
             " https://huggingface.co/settings/tokens"
         )
 
+    download_files = ["tokenizer*", "generation_config.json"]
+    if download_safetensors:
+        from safetensors.torch import load_file as safetensors_load
```

Comment:
you should move this import below to where you use it

Reply:
@carmocca Yes, I was thinking about this was well. But a counterexample is that if someone downloads the 180B model, it can possibly take an >2 hours. Then, it would be super frustrating if it fails later. So having it on top here makes it fail immediately.

On the other hand, I just notice that it would not redownload the files if they are there, so we can indeed move it down.